data-access/persoon.ts

```diff
@@ -204,3 +228,68 @@ export async function copyPerson(subject: Term, graph: Term) {
     throw Error(`Could not copy person with uri: ${escaped.person}`);
   }
 }
+
+async function getFractie(
+  id: string,
+  bestuursperiodeId: string,
+): Promise<TermProperty | null> {
+  const getQuery = `
+    PREFIX extlmb: <http://mu.semte.ch/vocabularies/ext/lmb/>
+    PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+    PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+    PREFIX org: <http://www.w3.org/ns/org#>
+    PREFIX person: <http://www.w3.org/ns/person#>
+    PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+
+    SELECT DISTINCT ?fractie
+    WHERE {
+      GRAPH ?graph {
+        ?persoon a person:Person;
+          mu:uuid ${sparqlEscapeString(id)};
+          extlmb:currentFracties ?fractie.
+
+        ?mandataris a mandaat:Mandataris;
+          mandaat:isBestuurlijkeAliasVan ?persoon;
+          org:holds ?mandaat.
+
+        ?mandaat ^org:hasPost ?bestuursorgaan.
```

Comment:
Not sure why you need these lines. Don't you just want to check the fracties that are set on the person for that bestuursperiode? It looks good, but I think you don't need to check the mandataris as well.